SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -63,6 +58,20 @@ void UGridBasedLBStrategy::Init()
 	}
 }
 
+void UGridBasedLBStrategy::SetLocalVirtualWorkerId(VirtualWorkerId InLocalVirtualWorkerId)
+{
+	if (!VirtualWorkerIds.Contains(InLocalVirtualWorkerId))
+	{
+		// This worker is simulating a layer which is not part of the grid.
+		LocalCellId = WorkerCells.Num();
```

Comment:
This seems like an arbitrary indicator for an invalid cell id? Can we have a constant in anonymous namespace at top of file and set it to something like -1?

Reply:
The problem is that it's an unsigned, and I'd rather not have to deal with signed/unsigned (yes, the overflow case is rather unlikely, but still). Even though it's a bit of a magic number, it's also always invalid, so even if the value was that because of a code error, we would want to early out.

Reply:
Personal pref would be improving readability (some individual functions below use `WorkerCells.Num()` and it seems a little magic in those individual circumstances) vs. making it signed, or even having an InvalidCellId local variable which is set to `WorkerCells.Num()` on initialisation?

Reply:
I like the extra variable, so I've added that.

Reply:
Ah sorry, I meant having an extra variable to indicate what the invalid value was, i.e. `InvalidCellId = WorkerCells.Num()`, and then having `check(LocalCellId != InvalidCellId)`. This also works but feels a little bit more like duplicated state vs. setting the invalid one. Either way, not something that blocks review 😄